Pipfile

```diff
@@ -12,7 +12,7 @@ lxml = ">=4.5.2"
 typing_extensions = ">=3.7"
 python-dateutil = ">=2.8.1"
 tzlocal = ">=2.1"
-pytz>=2019.3
+pytz = ">=2019.3"
```

Comment:
Please split the "mistakes" up and put them into the appropriate commits themselves. We wouldn't normally merge a commit that was like this. Let us know if you want help with git.

Reply:
Yup. Apologies for that :sweat_smile:. I'll put the responsible code changes in separate commits :)